test/function.localstack.test.ts

```diff
@@ -131,7 +100,7 @@ localstackTestSuite("functionStack", (testResource, _stack, _app) => {
   // eslint-disable-next-line no-only-tests/no-only-tests
   test.only = _testFunc(testResource.only);
 
-  test(
+  test.only(
```

Comment:
oops?

Reply:
Why didn't that get removed? strange... thanks for the catch.